src/modules/sv_detection_methods/analyze_sa_tag_method.cpp

```diff
@@ -47,6 +47,10 @@ void retrieve_aligned_segments(std::string const & sa_string, std::vector<Aligne
             int32_t mapq = std::stoi(fields[4]);
             aligned_segments.push_back(AlignedSegment{orientation, ref_name, pos, mapq, cigar_vector});
         }
+        else
+        {
+            seqan3::debug_stream << "Your SA tag has a wrong format (wrong amount of parameters): " << sa_tag << '\n';
```

Comment:
Can we just have here a throw `std::runtime_error`?

Reply:
Also is it even worth having this like this? The SA tag will be generated by alignment tools right, not by hand. So the odds of there being an error there would be basically none... So maybe it's more efficient to just take the if statement out.

Reply:
Hmmm, that is a good question. What do you think @eldariont ? I'm really not sure.

Reply:
Unfortunately, the odds of having a strange number of fields are not none. Last year, I was contacted by a user who reported that SVIM crashed on his data. When investigating the issue, I found that his chromosome names looked like this: `>Scaffold1;SomeOtherInfo=1`. The semi-colon in there caused the SA tags to be splitted into more than six fields. Therefore, I added the check to make sure that an error is raised as close to the problem as possible.
I don't mind whether we throw `std::runtime_error` or issue a debug message. The second option would be more gentle because it would tolerate a few malformed SA tags instead of terminating.

Reply:
Hm that's annoying hahaha. Then let's leave this as it is!